panels/job_status.py

```diff
@@ -186,7 +186,7 @@ def create_status_grid(self, widget=None):
                 self.heater_button[device].set_halign(Gtk.Align.START)
                 self.labels['temp_grid'].attach(self.heater_button[device], n, 0, 1, 1)
                 n += 1
-        extra_item = True
+        extra_item = not self._config.get_main_config().getboolean('show_heater_power', False)
```

Comment:
JobStatusPanel inherits from ScreenPanel :)
```suggestion
extra_item = not self._show_heater_power
```